cloudsync_onedrive.py

```diff
@@ -1282,6 +1308,11 @@ def _join_parent_reference_path_and_name(self, pr_path, name):
             if not found:
                 raise Exception("path '%s'(%s, %s) does not start with '%s', maybe implement recursion?" % (path, pr_path, name, preambles))
         path = urllib.parse.unquote(path)
+
+        if self._is_biz and self.namespace.is_shared:
```

Comment:
This code appears 3 times, but the other two versions use `self.namespace.shared_folder_path` instead of `self.namespace.is_shared`. Does this mean the same thing as the others? is this an opportunity to functionalize this?

Reply:
yup, they are the same -- pulled this out into a function.